custom_components/opnsense/update.py

```
@@ -263,71 +204,79 @@
-        client = self._get_opnsense_client()
-        task_details = client.upgrade_firmware(upgrade_type)
         sleep_time = 10
         running = True
         while running:
             time.sleep(sleep_time)
             try:
-                response = client.upgrade_status()
+                response = self._client.upgrade_status()
                 # after finished status is "done"
                 running = response["status"] == "running"
             except:
```

Ruff

Do not use bare `except`

(E722)